src/components/FeatureFlag/README.md

```diff
@@ -0,0 +1,28 @@
+To use the feature flag wrapper component, do the following:
+
+Example 1: making a new feature component that replaces loose content.
+<FeatureFlag feature_name='MyFeature'>
```

Comment:
how do we go about switching to new feature?
while keeping still old feature for a while in code
is there is a way or do we need to implement a `is_feature`